README.md

```diff
@@ -173,14 +173,18 @@ The following instructions will fetch Qt from your distribution's repositories i
 
 3. Clone repository
 
-    `git clone https://github.com/monero-project/monero-gui.git`
+    ```
+    git clone --recursive https://github.com/monero-project/monero-gui.git
+    cd monero-gui
+    ```
 
 4. Build
 
     ```
-    cd monero-gui
-    QT_SELECT=5 ./build.sh
+    make release -j4
     ```
+    \* `4` - number of CPU threads to use  
+    \* Add e.g. `CMAKE_PREFIX_PATH=$HOME/Qt/5.9.7/gcc_64/` enviroment variable to set a custom Qt install directory
```

Comment:
Could we slightly rephrase this to indicate that `CMAKE_PREFIX_PATH` should be prepended to `make` command. Or do you think the current wording is clear enough?

Reply:
Maybe like this?

```suggestion
    \* Add e.g. `CMAKE_PREFIX_PATH` enviroment variable to set a custom Qt install directory, e.g. `CMAKE_PREFIX_PATH=$HOME/Qt/5.9.7/gcc_64/  make release -j4`
```

Or do you have better wording?

Reply:
Looks good